app/main.py

```diff
@@ -1,8 +1,43 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
```

Comment:
This function  could be simplified
You can write only two loops: 
1. The first loop just creates all people without the husband/wife attribute
2. The second one will set them depending on values from dicts.

Try to simplify this code. If you have questions ask in the slack - we will discuss all of them there